spec/rails_admin/adapters/mongoid/property_spec.rb

```diff
@@ -207,7 +207,7 @@ class LengthValiated
 
     it 'does not cause problem with custom validators' do
       class MyCustomValidator < ActiveModel::Validator
-        def validate(_r); end
+        def validate(_record); end
```

Comment:
`Naming/UncommunicativeMethodParamName: Method parameter must be at least 3 characters long.`